pkg/sql/vars.go

```diff
@@ -229,6 +229,25 @@ var varGen = map[string]sessionVar{
 	// See https://www.postgresql.org/docs/9.6/static/runtime-config-client.html
 	`extra_float_digits`: nopVar,
 
+	// Supported for PG compatibility only.
+	// See https://www.postgresql.org/docs/9.4/static/runtime-config-client.html
```

Comment:
Not worth a CI run, but next time please grab the most recent version of the doc link.